### ISSUE TITLE
kiwix-plug should allow USB key hot pluging 

### ISSUE THREAD

**Issue by kelson42** [kiwix/plug#7]
If the data is available on a amovible flash drive then unplug it should stop the service and plugging it should restart it. This should allow to change the content without restarting the whole OS. Seb had done something (this attachment in the past) but after testing this still need to be fixed.
[kiwix-plug.raspberrypi.txt](https://github.com/kiwix/plug/files/981750/kiwix-plug.raspberrypi.txt)

Reported first here https://sourceforge.net/p/kiwix/feature-requests/955/

**Comment by qpkorr** [comment]
Nice idea.  For the record - I had another problem on the Raspberry Pi - it has no off button - so the only way to (safely) shut it down (in a remote location (I took it to a remote festival) - away from a computer) is to log in via an ssh client on my phone - which seemed clunky.  So instead - I made use of a script called unplug2shutdown.py - which detects a usb key being unplugged, and initiates a shutdown.  I changed the usb key to be mounted read-only to ensure no ill-effects to the key.  The changes are in my branch here https://github.com/qpkorr/plug/tree/qpkorr_rpi .  Your idea may well be better, but figured I should add this one to the mix.
